src/marketo/types.d.ts

```diff
@@ -3,5 +3,6 @@ export interface MarketoFormObject {
   getFormElem(): HTMLFormElement[];
   validate(): boolean;
   submit(): MarketoFormObject;
+  reset(): MarketoFormObject;
```

Comment:
@georgeebili 
This method doesn't exist https://developers.marketo.com/javascript-api/forms/api-reference/

Reply:
Ooo! I taught you where the one who wrote the Script.
Because document.selector(Form).reset() and 
document.selector(Form).submit() does exist.
Well I will have to develop the solution via JS.